### PR TITLE
Change html end tag to have the same color as the start tag

### PR DESCRIPTION
**Pull request by tonyjmnz**

### CODE REVIEW

File: colors/monokai.vim [hunk 1]
```diff
@@ -87,7 +87,7 @@ hi erubyDelimiter ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui
 hi erubyComment ctermfg=95 ctermbg=NONE cterm=NONE guifg=#75715e guibg=NONE gui=NONE
 hi erubyRailsMethod ctermfg=81 ctermbg=NONE cterm=NONE guifg=#66d9ef guibg=NONE gui=NONE
 hi htmlTag ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
-hi htmlEndTag ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
+hi htmlEndTag ctermfg=NONE ctermbg=NONE cterm=NONE guifg=#a6e22e guibg=NONE gui=NONE
```
Comment (sickill) [inline colors/monokai.vim]: Hmm.. `htmlTag` has `guifg=NONE` so you're not setting it to the same value here. Am I correct?

Comment (tonyjmnz) [inline colors/monokai.vim]: Yeah I know it looks awkward, however, it seems like the HTML start tag "inherited" (if that's the right word in this context) its color from `Function`, however, the HTML end tag didn't.

#### Original config:
screen shot 2015-08-03 at 11 07 44 am

#### Proposed change:
screen shot 2015-08-03 at 11 06 25 am

Comment (sickill) [inline colors/monokai.vim]: Ah, ok. It's weird indeed, but if it works then :+1:

**Comment by joeytwiddle** [comment]
Oh, I just noticed that my latest PR undid this. Sorry about that!

If we want `<` and `>` to be green, then lets explicitly set that, in `ctermfg` as well as `guifg`, and for both highlight groups.

(Alternatively, we can explicitly inherit from Function for both groups. But that might require clearing the highlight groups first, and it would probably violate [Least Astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) for users.)

**Pull request merged**